tracer/test/Datadog.Trace.Tests/Snapshots/PublicApiTests.PublicApiHasNotChanged.verified.txt

```diff
@@ -140,10 +138,18 @@ namespace Datadog.Trace.Configuration
         public bool StartupDiagnosticLogEnabled { get; }
         public bool TraceEnabled { get; }
         public bool TracerMetricsEnabled { get; }
+        public Datadog.Trace.Configuration.ImmutableExporterSettings Exporter { get; }
+        public static Datadog.Trace.Configuration.ImmutableTracerSettings FromDefaultSources() { }
+    }
+    public class ImmutableExporterSettings
+    {
+        public ImmutableExporterSettings(Datadog.Trace.Configuration.IConfigurationSource source) { }
+        public ImmutableExporterSettings(Datadog.Trace.Configuration.ExporterSettings settings) { }
+        public System.Uri AgentUri { get; }
+        public string MetricsPipeName { get; }
+        public Datadog.Trace.Configuration.TraceTransportType TraceTransport { get; }
```

Comment:
nit: Since we have properties with the `Traces` prefix can we apply that to this property and the type name?